test/lit/passes/multi-memory-lowering.wast

```diff
@@ -628,6 +628,182 @@
    )
   )
   )
+
+  ;; CHECK:      (func $memory.fill
+  ;; CHECK-NEXT:  (memory.fill
+  ;; CHECK-NEXT:   (i32.const 0)
+  ;; CHECK-NEXT:   (i32.const 1)
+  ;; CHECK-NEXT:   (i32.const 2)
```

Comment:
The order of these three instructions (0, 1, 2) should remain the same (imagine they are three calls, that have side effects, so they have to keep running in the same order). But atm the order in the output for bounds checks looks like `2, 0, 1`.

If there isn't a better way, a simple approach for stuff like this is to allocate 3 local vars up front and write the inputs to them immediately, and afterwards use new LocalGets of their indexes.